Cargo.toml

```
@@ -27,7 +27,7 @@ miniz_oxide.opt-level = 3
 [profile.release]
 incremental = true
 # Set this to 1 or 2 to get more useful backtraces in debugger.
-debug = 0
+debug = 2
```

```suggestion
debug = 0
```
(reminder)
also for this you could use the `dev-rel` profile